app/models/listing.rb

```diff
@@ -19,4 +19,12 @@ class Listing < ApplicationRecord
   def full_address
     [address, city, state, zipcode, country].compact.join(', ')
   end
+
+  def editable_by?(user)
+    return true if user.role == 'admin'
```

Comment:
are we doing caps or lower case for roles? I think I've seen 'Admin' somewhere else.